web-common/src/features/dashboards/leaderboard/DimensionLeaderboardEntrySet.svelte

```diff
@@ -115,10 +116,10 @@ see more button
       {excluded}
     >
       <svelte:fragment slot="label">
-        {label}
+        <FormattedDataType isNull={label === null} value={label} />
```

Comment:
Just curious: are there scenarios where you can have a valid value being passed in to FormattedDataType / Number, but also have isNull being true?

In both instances of using isNull that I see in this PR, it is directly tied to the value, albeit it in two different ways:
```svelte
// value is label, isNull is whether value === null
<FormattedDataType isNull={label === null} value={label} /> 

// value is either of formattedValue or value; if either of those is false-y, mark as null
<Number type="INTEGER" isNull={!(formattedValue || value)} value={formattedValue || value} /> // 
```

If the logic for determining isNull is always directly dependent on the value being passed in, could the isNull logic be unified and live inside these components so that you only have to pass the value?

Also, in the number example:
```svelte
<Number
          type="INTEGER"
          isNull={!(formattedValue || value)}
          value={formattedValue || value}
/>
```

What if `formattedValue` is undefined and `value` is explicitly 0? This would still mark the value as being null, as `!(undefined || 0)` === true.

Reply:
Good catch with value being 0. Will fix that.
The `isNull` prop is also being used differently at different places. Sometimes if we have an empty string, we want the null styling so it's useful in those cases. Also I don't think we need to always pass the `isNull` prop. This https://github.com/rilldata/rill-developer/blob/b3800c8749257f62481cbed63abf48854e3058c8/web-common/src/components/data-types/FormattedDataType.svelte#L43

takes care of `isNull` prop inside the component if the value is null so I can probably get rid of the isNull prop. Thanks!